compiler/rustc_session/src/options.rs

```diff
@@ -904,6 +904,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "force all crates to be `rustc_private` unstable (default: no)"),
     fuel: Option<(String, u64)> = (None, parse_optimization_fuel, [TRACKED],
         "set the optimization fuel quota for a crate"),
+    function_sections: Option<bool> = (None, parse_opt_bool, [TRACKED],
+        "whether each function should go in its own .text section"),
```

Comment:
`.text` is not exactly the most portable name (e.g. machO uses `__text` AFAIR). I suggest dropping the `.` or perhaps the whole `.text`.

Reply:
That's a good point, I removed it.